src/platform/site-wide/sass/shame.scss

```diff
@@ -571,3 +571,27 @@ img[data-srcset] {
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 }
+
+.no-p-bottom-margin p {
+  margin-bottom: 0;
+}
+
+.operating-status-title {
+  margin: auto;
```

Comment:
`.vads-u-margin-x--auto` might cover this

Reply:
I tried this first, but it didn't work the way I need it to, @rianfowler

Reply:
Unfortunately, it doesn't position the title elements correctly, so need to use margin: auto, @rianfowler